src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -114,10 +114,11 @@ System.Resources.Extensions.PreserializedResourceWriter</PackageDescription>
                       Lines="$(SuggestedBindingRedirectsPackageFileContent)"
                       Overwrite="true" />
 
-    <ItemGroup>
-      <Content Include="$(SuggestedBindingRedirectsPackageFile)"
-               PackagePath="buildTransitive\$(NetFrameworkMinimum)\$(PackageId).targets" />
-    </ItemGroup>
   </Target>
 
+  <ItemGroup Condition="'$(NetFrameworkMinimum)' != ''">
```

Comment:
Should this condition validate that the project actually targets this TFM? I.e. TargetFrameworks.Contains(...).

Reply:
I copied the existing condition that was already present on the Target.  I think it's there to support the case where `NetFrameworkMinimum` is not defined.  I think it's a bit redundant to have `TargetFrameworks.Contains` - in either case we'd want to *delete* this item if the project no longer targets `NetFrameworkMinimum` and I don't think a contains check helps us catch that any better.